controllers/selfnoderemediation_controller.go

```diff
@@ -53,6 +54,8 @@ var (
 
 	lastSeenSnrNamespace  string
 	wasLastSeenSnrMachine bool
+
+	untreatableError = errors.New("untreatable error, stop reconciling")
```

Comment:
I think I would prefer a custom error type, instead of relying on unwrapping exactly once to find this error message